mesh_aws_client/tests/mesh_check_send_parameters_application_test.py

```diff
@@ -273,23 +285,23 @@ def sample_trigger_event():
                 "bucketName": "meshtest-mesh",
                 "X-Amz-Algorithm": "AWS4-HMAC-SHA256",
                 "x-amz-acl": "private",
-                "X-Amz-SignedHeaders": "content-md5;content-type;host;x-amz-acl;x-amz-storage-class",  # noqa pylint: disable=line-too-long
```

Comment:
Any change here was me trying to fix something an auto-reformater accidently did. Does removing this space make a difference?